app/controllers/dashboard_controller.rb

```diff
@@ -5,14 +5,20 @@ class DashboardController < ApplicationController
   skip_verify_authorized only: :show
 
   def show
-    # This is going to be slow because it makes a call to DSA for each work.
-    # It is also going to block rendering since it is synchronous.
+    # This is going to be slow because it is going to block rendering since it is synchronous.
     # Also, it processes all of a user's works, but not all may be rendered on the dashboard.
+    @druid_to_status_map = Sdr::Repository.statuses(druids: druids_for_works_for_current_user)
```

Comment:
If this instance variable isn't used in any views, it could be a local variable so it doesn't leak.

Reply:
Agree. `@` was cruft.